packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -354,7 +354,7 @@ const BaseButton = ({
         flexDirection="row"
         alignItems="center"
         justifyContent="center"
-        alignSelf="center"
+        flex={1}
```

Comment:
Need to specify the flex for the content other native would not take up the whole real-estate available within button and restrict content to the top

Reply:
Why is this happening suddenly? Previously it was fine.

Reply:
The PR was open for too long, I must've missed checking this on native 🤦‍♂️